DirectProgramming/C++SYCL_FPGA/Tutorials/Features/max_reinvocation_delay/src/max_reinvocation_delay.cpp

```diff
@@ -0,0 +1,86 @@
+#include <iostream>
+#include <sycl/ext/intel/fpga_extensions.hpp>
+#include <sycl/sycl.hpp>
+
+#include "exception_handler.hpp"
+
+#define FACTORS 5
```

Comment:
FACTORS should follow convention used in other samples of constexpr (not #define).

Reply:
Now addressed.